modules/lfs/content_store.go

```diff
@@ -53,6 +53,10 @@ func (s *ContentStore) Get(meta *models.LFSMetaObject, fromByte int64) (io.ReadC
 	}
 	if fromByte > 0 {
 		if fromByte >= meta.Size {
+			err = f.Close()
+			if err != nil {
+				log.Error("Whilst trying to read LFS OID[%s]: Unable to close Error: %v", meta.Oid, err)
```

Comment:
```suggestion
				log.Error("Whilst trying to read LFS OID[%s]: Unable to close: %v", meta.Oid, err)
```

Reply:
> Not very familiar with this code, but shouldn't we close in the other error-cases too? (`Unable to seek`)

Yup it's gonna need that too

Reply:
The rest of log messages in that function look like this - so if you want to clean this up - you need to clean them all up.